src/components/book.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ book: { id, title, category } }) => (
+  <tr>
+    <td>{ id }</td>
+    <td>{ title }</td>
+    <td>{ category }</td>
+  </tr>
+);
+
+Book.propTypes = {
+  book: PropTypes.instanceOf(Object).isRequired,
```

Comment:
using PropTypes.shape here will allow you set a type check that is more accurate